### ISSUE TITLE
Timeout is still an invalid value

### ISSUE THREAD

**Issue by rockbenben** [SamKirkland/FTP-Deploy-Action#318]
**Bug Description**
I set `timeout` to 10 minutes, but the action ran for less than 4 minutes and an error timeout was reported.
Does `timeout` not take effect?

**My Action Config**
```yaml
      - name: 📂 Sync files
        uses: SamKirkland/FTP-Deploy-Action@4.3.2
        with:
          local-dir: docs/.vuepress/dist/
          server: ${{ secrets.ftp_host }}
          username: ${{ secrets.ftp_username }}
          password: ${{ secrets.ftp_password }}
          port: ${{ secrets.ftp_port }}
          timeout: 600000
```

**My Action Log**
```
Error: Timeout (control socket)
    at Socket.<anonymous> (/home/runner/work/_actions/SamKirkland/FTP-Deploy-Action/4.3.2/dist/index.js:5288:58)
    at Object.onceWrapper (events.js:420:28)
    at Socket.emit (events.js:314:20)
    at Socket._onTimeout (net.js:483:8)
    at listOnTimeout (internal/timers.js:554:17)
    at processTimers (internal/timers.js:497:7)
Error: Error: Timeout (control socket)
```
![image](https://user-images.githubusercontent.com/28252913/188327776-61b9928e-f148-4b0f-8c45-9c4627177712.png)

**Comment by kubaTro26** [comment]
i have timeout error too

**Comment by SamKirkland** [comment]
I believe this was an issue with an underlying library (basic-ftp) which was fixed in [basic-ftp v5.0.2]([5.0.2](https://github.com/patrickjuchli/basic-ftp/releases/tag/v5.0.2)) "Fixed: Destroy sockets explicitly on timeouts only."

`v4.3.4` of this action uses the latest version of basic-ftp, closing for now we can reopen if its still an issue

**Comment by jeremyolu** [comment]
I am getting this issue - 
Unable to resolve action `SamKirkland/FTP-Deploy-Action@4.3.4`, unable to find version `4.3.4`

I have updated from version 2 to 3 - actions/checkout@v3

updated SamKirkland/FTP-Deploy-Action@4.3.2 to 4.3.4

**Comment by rockbenben** [comment]
> I am getting this issue - Unable to resolve action `SamKirkland/FTP-Deploy-Action@4.3.4`, unable to find version `4.3.4`
> 
> I have updated from version 2 to 3 - actions/checkout@v3
> 
> updated SamKirkland/FTP-Deploy-Action@4.3.2 to 4.3.4
> 
> but still getting this issue

The issue hasn't occurred again. The new name is `SamKirkland/FTP-Deploy-Action@v4.3.4`.

**Comment by AliSafaei1990** [comment]
had same issue... 
just changed a file on my project and did a push again. now it is working fine!

**Comment by jsqrt** [comment]
`uses: SamKirkland/FTP-Deploy-Action@v4.3.5`

I think I fixed it by adding a 'v' before v4.3.5/. 

Thanks @SamKirkland, awesome package ^-^
